network/proxy.md

```diff
@@ -11,32 +11,30 @@ configure it in different ways:
   [configure the Docker client](#configure-the-docker-client) to pass
   proxy information to containers automatically.
 
-- In Docker 17.06 and lower, you must
-  [set appropriate environment variables](#use-environment-variables)
+- In Docker 17.06 and earlier versions, you must set the appropriate
```

Comment:
@StefanScherer Not sure if we should still make this distinction as 17.06 is quite old.